apps/web/app/(use-page-wrapper)/(main-nav)/availability/page.tsx

```diff
@@ -25,10 +27,14 @@ const Page = async () => {
     //   orgId,
     //   userId,
     // });
+    const t = await getTranslate();
     return (
-      <AvailabilityPage
-      //  currentOrg={currentOrg}
-      />
+      <ShellMainAppDir
+        heading={t("availability")}
+        subtitle={t("configure_availability")}
```

Comment:
fyi, `title` or `description` are not needed. They are passed to `Shell.tsx` for SEO purposes, but they aren't used anywhere in `ShellMain`. Also, we turn `withoutSeo={true}` anyways since App Router handles Metadata already